4.4.0/nginx/default.conf.template

```diff
@@ -0,0 +1,28 @@
+server {
+  listen ${NGINX_PORT};
+  server_name ${NGINX_HOST};
+  
+  access_log /var/log/nginx/gn-access.log;
+  error_log /var/log/nginx/gn-error.log;
```

Comment:
@juanluisrp Should we add ` client_max_body_size 100M;` to allow upload because with default setting importing a ZIP of 1.5Mo is failing ?